### PR TITLE
 main/glib: upgrade to 2.62.0

### PR DESCRIPTION
**Pull request by Cogitri**
* Adopt maintainership

### CODE REVIEW

**Comment by Cogitri** [comment]
Could someone else maybe test Qemu? I don't have VMs.

File: main/glib/APKBUILD [hunk 1]
```diff
@@ -10,7 +11,7 @@ license="LGPL-2.1-or-later"
 triggers="$pkgname.trigger=/usr/share/glib-2.0/schemas:/usr/lib/gio/modules"
 depends_dev="python3 gettext-dev bzip2-dev libxml2-utils libxslt
 	docbook-xml docbook-xsl"
-makedepends="$depends_dev pcre-dev meson ninja
+makedepends="$depends_dev pcre-dev meson dbus-dev dbus
```
Comment (ncopa) [inline main/glib/APKBUILD]: `dbus-dev` here will introduce a circular dependency since dbus needs `glib-dev` to build.

Comment (Cogitri) [reply main/glib/APKBUILD]: Oh right, thanks for noticing

**Comment by ncopa** [comment]
> Could someone else maybe test Qemu? I don't have VMs.

builds. :+1:

File: main/glib/APKBUILD (hunk 1 above)
Comment (ncopa) [reply main/glib/APKBUILD]: why is it needed? it seems to build fine without dbus.

**Comment by Cogitri** [comment]
Great thanks. Will have to fix this crashing gnome-shell, somehow they break something about this every minor release...

File: main/glib/APKBUILD (hunk 1 above)
Comment (Cogitri) [reply main/glib/APKBUILD]: Ah, it's for additional tests, but we can live without that to avoid the dependency loop.

**Comment by Cogitri** [comment]
Alrighty, this works with the gjs release candidate. Would it be OK to upgrade to that already, or should we wait for the actual stable release? I'd personally prefer using the gjs release candidate just so we have a more smooth rollout of GNOME since otherwise almost all of it will block on this which would be unfortunate and the gjs release should be done by September 11 (hopefully sooner).

**Comment by Cogitri** [comment]
There's no change in 1.58.0 (other than the version number). I'll bump qemu for this and than this should he good to go.

**Comment by Cogitri** [comment]
Mind taking another look, @ncopa?

**Comment by Cogitri** [comment]
Qemu fails:

```
usr/lib/gcc/i586-alpine-linux-musl/8.3.0/../../../../i586-alpine-linux-musl/bin/ld: cannot find -lz
```

Not quite sure why since you said that it works for you though and zlib-dev is a dep (so I guess it might be on linking order?)

**Comment by Cogitri** [comment]
Removed Qemu for now, we can bump it later but I don't really want to block GNOME on this :/

**Pull request closed without merging**